operators/multiclusterobservability/controllers/placementrule/customize_img.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2021 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package placementrule
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	imageregistryv1alpha1 "github.com/open-cluster-management/multicloud-operators-foundation/pkg/apis/imageregistry/v1alpha1"
+	"github.com/open-cluster-management/multicloud-operators-foundation/pkg/helpers/imageregistry"
+)
+
+var (
+	managedClusterImageRegistry = map[string]string{}
+)
+
+func updateManagedClusterImageRegistry(obj client.Object) {
+	if imageReg, ok := obj.GetLabels()[imageregistryv1alpha1.ClusterImageRegistryLabel]; ok {
```

Comment:
do we need to add "delete" logic here?
Assume one managed cluster has custom registry firstly, after some time, delete it.

Reply:
@zhiweiyin318 Can we update it automatically when disabling custom registry from managedcluster? I remember that seems we can not update it automatically.